events/class_tests/test_create_event_views.py

```diff
@@ -0,0 +1,58 @@
+import pytest
```

Comment:
Can you change the name of this file to:
"create_event_views_test.py"

Reply:
done